tests/export/html/test_heading.py

```diff
@@ -340,9 +348,16 @@ def test_heading_in_list_with_bare_paragraph(self):
         document.add(MainDocumentPart, document_xml)
 
         expected_html = '''
-            <h1>foo</h1>
-            <p>bare paragraph</p>
-            <h1>bar</h1>
+            <ol class="pydocx-list-style-type-decimal">
+                <li>
+                    <strong>foo</strong>
+                    <br />
+                    bare paragraph
```

Comment:
I don't understand why this is happening. Shouldn't this be two lists separated by a paragraph? That is closer to what the XML is showing happening.